docker/mysql/db/db_upgrade/portal/Update_RMC_DB_2023030900-2023040700.sql

```diff
@@ -0,0 +1,24 @@
+START TRANSACTION;
+
+ CREATE TABLE `template_history_files` (
+   `file_id` int(11) NOT NULL AUTO_INCREMENT,
+   `file_parent_name` text,
+   `file_name` text,
+   `file_path` text,
+   `file_size` mediumint(9) DEFAULT NULL,
+   `file_modify_by` text,
+   `file_created` text,
+   PRIMARY KEY (`file_id`)
+ ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+ UPDATE `settings` SET `data` = '2023040600' WHERE `settings`.`setting` = 'dbversion';

```

Comment:
your db file is labeled 2023040700 make sure this matches

Reply:
Have made the changes.